package.json

```diff
@@ -21,10 +21,13 @@
     "@protobuf-ts/grpc-transport": "^2.9.0",
     "@protobuf-ts/runtime-rpc": "^2.9.0",
     "@swc/helpers": "0.5.3",
+    "@types/imurmurhash": "^0.1.3",
+    "@types/semver": "^7.5.4",
```

Comment:
all `@types...` packages should be DEV dependencies.

Reply:
@toddbaert any reason for this ? 🤔

Reply:
This is done anyway

Reply:
This is because types are only used at compile-time, not runtime. Anything that is only used at compile-time is a devDependency - there's no need to ship them or force consumers to download them.